ui/imports/shared/views/TransactionDetailView.qml

```diff
@@ -212,23 +212,27 @@ Item {
                     id: baseFee
                     maxWidth: parent.width
                     primaryText: qsTr("Base fee")
-                    secondaryText: root.transaction !== undefined && !!root.transaction ?  qsTr("%1 Gwei").arg(RootStore.hex2Gwei(root.transaction.baseGasFees)) : ""
+                    secondaryText: root.isTransactionValid ?  qsTr("%1").arg(LocaleUtils.currencyAmountToLocaleString(root.transaction.baseGasFees)) : ""
                 }
                 InformationTile {
                     maxWidth: parent.width
                     primaryText: qsTr("Tip")
-                    secondaryText: root.transaction !== undefined && !!root.transaction ?  qsTr("%1 Gwei <font color=\"#939BA1\">&#8226; Max: %2 Gwei</font>").
-                                                                    arg(RootStore.hex2Gwei(root.transaction.maxPriorityFeePerGas)).
-                                                                    arg(RootStore.hex2Gwei(root.transaction.maxFeePerGas)) : ""
+                    secondaryText: root.isTransactionValid ?  "%1 <font color=\"%2\">&#8226; %3: %4</font>".
+                                                                    arg(LocaleUtils.currencyAmountToLocaleString(root.transaction.maxPriorityFeePerGas)).
+                                                                    arg(Theme.palette.baseColor1).
+                                                                    arg(qsTr("Max")).
+                                                                    arg(LocaleUtils.currencyAmountToLocaleString(root.transaction.maxFeePerGas)) : ""
                     secondaryLabel.textFormat: Text.RichText
                 }
             }
             InformationTile {
                 maxWidth: parent.width
                 primaryText: qsTr("Total fee")
-                secondaryText: root.transaction !== undefined && !!root.transaction ? qsTr("%1 Gwei <font color=\"#939BA1\">&#8226; Max: %2 Gwei</font>").
-                                                                arg(Utils.stripTrailingZeros(RootStore.hex2Gwei(root.transaction.totalFees))).
-                                                                arg(Utils.stripTrailingZeros(RootStore.hex2Gwei(root.transaction.maxTotalFees))) : ""
+                secondaryText: root.isTransactionValid ? "%1 <font color=\"%2\">&#8226; %3: %4</font>".
+                                                                arg(LocaleUtils.currencyAmountToLocaleString(root.transaction.totalFees)).
+                                                                arg(Theme.palette.baseColor1).
+                                                                arg(qsTr("Max")).
```

Comment:
Would be better to keep `qsTr("Max: %1)`; same for the `qsTr("on")` string above

It's hard to translate "on" by itself; impossible to imagine for the translators to figure out what the standalone "on" is used and in which context

Reply:
Interesting... I thought translators were able to easily get some context and know where the strings they're translating come from. Allright, I'll make that change.

Reply:
it's impossible :)

The string "on" would end up in the TS files only once. Now imagine you have 2 different contexts:

```qml
StatusSwitch {
   text: checked ? qsTr("on") : qsTr("off")
}

StatusBaseText {
   text: "%1 %2 %3".arg(foo).arg(qsTr("on")).arg(bar)
}
```

It's obvious that "on" means completely different things here in these 2 cases

Reply:
Yeah, I was kind of expecting the translation mechanism to be friendlier and give translators some extra context (file + line number, whole line where the `qsTr` call was made) instead of just the string contained inside the `qsTr` call, to disambiguate in such cases.

Reply:
Such context (file/line number) is provided but still this is not enough. Both occurences will be merged to just one string to be translated

Reply:
BTW, `on` is directly surrounded by the HTML tag in this case:

```
                   secondaryText:  root.transaction !== undefined && !!root.transaction ? qsTr("%1 <font color=\"#939BA1\">on</font> %2").
                                                                     arg(Utils.formatShortTime(transaction.timestamp * 1000, RootStore.accountSensitiveSettings.is24hTimeFormat)).
                                                                     arg(Utils.formatShortDate(transaction.timestamp * 1000, RootStore.accountSensitiveSettings.is24hTimeFormat)): ""
```
What do you suggest here?

Reply:
I don't have a good/better idea :/ It has to stay as-is I'm afraid

Reply:
Allright! Left the `on` string as it was originally, made the suggested changes to the `Max` string.